README.md

```diff
@@ -191,6 +191,33 @@ A user-oriented guide describing how to share case and variant data to MatchMake
 be found [here][matchmaker-scout-sharing].
 
 
+## Development
+
+To keep the code base consistent, formatting with [Black](https://github.com/psf/black) is enforced.
```

Comment:
Enforce is a forceful word.😉 How about adding another sentence, like so:
```suggestion
To keep the code base consistent, formatting with [Black](https://github.com/psf/black) is always applied as part of the PR submission process via GitHub Actions. While not strictly required, to avoid confusion, it is suggested that developers apply Black locally.
```

Reply:
Good suggestion, file edited!